### PR TITLE
add deploymgr stuff

### PR DESCRIPTION
**Pull request by RLuckom**

### CODE REVIEW

File: profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl [hunk 1]
```diff
@@ -5,19 +5,34 @@ variable "google_project" {
...
 variable "vault_path_prefix" {
   default = "{{env "VAULT_PATH_PREFIX"}}"
   description = "The prefix path at which to store vault values"
 }
 
+variable "billing_accounts" {
+  default = [
+    "billing@test.firecloud.org",
+    "billing@quality.firecloud.org"
+  ]
```
Comment (RLuckom) [inline profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl]: probably should be ctmpl parameterized, not sure how to do so with an array, could just split it. thoughts?

File: profiles/rawls-sa/terraform/rawls/sa.tf [hunk 2]
```diff
@@ -11,6 +11,20 @@ resource "google_project_iam_member" "app" {
...
 
+resource "google_project_iam_member" "app-deployment-manager" {
+  count   = "${length(var.app_sa_roles)}"
+  project = "${var.deployment_manager_google_project}"
+  role    = "${element(var.app_sa_deployment_manager_roles, count.index)}"
+  member  = "serviceAccount:${google_service_account.app.email}"
+}
+
+resource "google_project_iam_member" "billing-deployment-manager" {
+  count   = "${length(var.billing_accounts)}"
```
Comment (helgridly) [inline profiles/rawls-sa/terraform/rawls/sa.tf]: nit: these are groups, not individual accounts

File: profiles/rawls-sa/terraform/rawls/sa.tf [hunk 3]
```diff
@@ -28,3 +42,18 @@ resource "vault_generic_secret" "app_account_key" {
   path = "${var.vault_path_prefix}/${var.service}/${var.service}-account.json"
   data_json = "${base64decode(google_service_account_key.app_account_key.private_key)}"
 }
+
+resource "google_service_account" "billingprobe" {
+  account_id   = "${var.owner}-billingprobe"
```
Comment (helgridly) [inline profiles/rawls-sa/terraform/rawls/sa.tf]: note to self: in the terraformy version of the rawls conf, the config gets the SA email address from the SA key checked into vault by terraform

File: profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl [hunk 4]
```diff
@@ -5,19 +5,34 @@ variable "google_project" {
   default = "{{env "GOOGLE_PROJECT"}}"
   description = "The google project as specified in the application json"
 }
+
+variable "deployment_manager_google_project" {
+  default = "{{if (env "DEPLOYMENT_MANAGER_PROJECT")}}{{env "DEPLOYMENT_MANAGER_PROJECT"}}{{else}}{{env "GOOGLE_PROJECT"}}-deployment-mgr{{end}}"
```
Comment (helgridly) [inline profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl]: can we change this to be `terra-deployments-{envname}`

File: profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl [hunk 5]
```diff
@@ -5,19 +5,34 @@ variable "google_project" {
...
   default = "{{env "SERVICE"}}"
   description = "The name of the service within the profile"
 }
+
 variable "vault_path_prefix" {
   default = "{{env "VAULT_PATH_PREFIX"}}"
   description = "The prefix path at which to store vault values"
 }
 
+variable "billing_accounts" {
```
Comment (helgridly) [inline profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl]: in rawls we call this the "billing service account" (even though it's not actually a service account)

File: profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl (hunk 1 above)
Comment (gmalkov) [reply profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl]: Yeah I guess we can just assume that if it's passed in as an env var it's a comma-separated list and split it 🤷‍♂

File: profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl (hunk 4 above)
Comment (RLuckom) [reply profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl]: done

File: profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl (hunk 1 above)
Comment (RLuckom) [reply profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl]: good point, I did that.

**Comment by RLuckom** [comment]
Created the deployment manager projects for the malkov and rluckom envs and added boverrides to the jsons

**Pull request merged**